### ISSUE TITLE
Feature requests: add song to multiple editions 

### ISSUE THREAD

**Issue by buges** [UltraStar-Deluxe/USDX#559]
Hi

Request for songs to be added to multiple editions

Maybe using a new identifier #EDITION2: etc.. or modifying the #EDITION tag to read editions separated by comma's
ie best of pop,hits of the 80's etc...

This would save on disk space by not having the same songs just with a different edition tag.

**Comment by basisbit** [comment]
you could just create a couple of copies of the .txt file - one .txt per version.

**Comment by buges** [comment]
Thanks

I never thought of that, You learn something new everyday.

**Comment by buges** [comment]
Hi

I did what you suggested and created copies of the text file each with different editions in them, but i've found that when browsing though the songs I've now got x amount of copies showing up as they are being picked up as new songs not the same song just with a different edition in the tag info.

**Comment by basisbit** [comment]
yes, but at least you can now find them if edition folder view is enabled. For anything else, you'll have to wait for someone to change the code.

**Comment by basisbit** [comment]
@buges could you please describe expected behavior for when the feature is implemented? Apparently a song should still only show up once. So what do you want changed, where should a song show up but doesn't show up currently?

**Comment by buges** [comment]
Hi

The expected behaviour when making multiple copies of a song's txt file and having a different edition in each one should be displayed as different editions (which is working fine) the problem is each txt file is displayed as x amount of separate song's when in other sorting modes. 

Easiest way to recreate this bug is:
copy a txt file, change the edition.

Now when you list the songs you have 2 copies of the same song in all sort modes except from sort by edition.

Expected behaviour should be just one copy of the song in all sort modes except sort by edition.

**Comment by basisbit** [comment]
just to be clear, this behavior is not a bug, it is as designed. The multi-edition support depending on grouping / sorting mode is a feature request. 😃

**Comment by buges** [comment]
Sorry I thought that multiple copies of the same song just with different edition tags showing up in all other sort modes expect sort by edition was a bug.

I've just duplicated a txt and changed nothing and got 2 copies of that song in the list, So it seems the array that scans the files adds them blindly just based on having a different file name, not the info that's in it.

Unfortunately this is beyond my skills to sort out.
